pkg/infra/iac2/templates_compiler.go

```diff
@@ -26,7 +26,7 @@ type (
 		// templates is the fs.FS where we read all of our `<struct>/factory.ts` files
 		templates fs.FS
 		// resourceGraph is the graph of resources to render
-		resourceGraph *graph.Directed[core.Resource] // TODO make this be a core.ResourceGraph, and un-expose that struct's Underlying
+		resourceGraph *core.ResourceGraph // TODO make this be a core.ResourceGraph, and un-expose that struct's Underlying
```

Comment:
should rm the TODO comment now:

```suggestion
		resourceGraph *core.ResourceGraph
```